grails-app/services/org/broadinstitute/orsp/PermissionService.groovy

```diff
@@ -10,31 +10,61 @@ class PermissionService implements UserInfo {
     UserService userService
 
     // get issue's collaborators as a List<String>
-    List<String> getIssueCollaborators(Issue issue) {
-      issue.extraProperties.findAll ({ it.name == IssueExtraProperty.COLLABORATOR }).collect { property -> property.value }
+    List<String> getIssueCollaborators(Map<String, List<String>> extraProperties) {
+        List<String> results = new ArrayList<String>()
+
+        def collaboratorsValues = extraProperties.findAll ({ it.key == IssueExtraProperty.COLLABORATOR }).values()
+        if (collaboratorsValues.size() != 0) {
+            results = collaboratorsValues.first()
+        }
+
+        collaboratorsValues = extraProperties.findAll ({ it.key == IssueExtraProperty.COLLABORATORS }).values()
+        if (results.size() == 0 && collaboratorsValues.size() != 0) {
+            results = collaboratorsValues.first()
+        }
+
+        results
     }
 
     // get issue's pms as a List<String>
-    List<String> getIssuePMs(Issue issue) {
-      issue.extraProperties.findAll ({ it.name == IssueExtraProperty.PM }).collect { property -> property.value }
+    List<String> getIssuePMs(Map<String, List<String>> extraProperties) {
+        List<String> results = new ArrayList<String>()
+        def collaboratorsValues = extraProperties.findAll ({ it.key == IssueExtraProperty.PM }).values()
+        if (collaboratorsValues.size() != 0) {
+            results = collaboratorsValues.first()
+        }
+        results
```

Comment:
Here again, why are we only showing a single one? The method name, return type, and comments indicate very different behavior.